src/nxdoc/web-ui/web-ui-tutorials/web-ui-configuration-properties.md

```diff
@@ -0,0 +1,183 @@
+
+     <!-- Properties to be fetched when loading current document, default 'properties' means all -->
+    <property name="org.nuxeo.web.ui.fetch.document" list="true">properties</property>
+    
+  </extension>
+</component>
+```
+
+### Date Formats
+Since Nuxeo Web UI is an internationalized and localized application, there is a concern with language and culture. For this reason, it is possible to configure some date and time related formats.
+
+By default, the `dateFormat` and `dateTimeFormat` are defined according to Moment.js syntax (see more [here](https://momentjs.com/)), since it is the library used by Web UI to manage time.
+Also, a `firstDayOfWeek` is considered in order to display a date picker, for example, according to the cultural needs. By not having that variable defined, it will consider the most common case that is Sunday. An example of its usage can be found on [Nuxeo Date Picker](https://github.com/nuxeo/nuxeo-elements/blob/master/ui/widgets/nuxeo-date-picker.js#L86-L100).
+
+```xml
+<component name="org.nuxeo.web.ui.properties.contrib">
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+
+     <!-- Date (time) format -->
+    <property name="org.nuxeo.web.ui.dateFormat">LL</property>
+    <property name="org.nuxeo.web.ui.dateTimeFormat">LLL</property>
+
+    <!-- First Day Of Week -->
+    <property name="org.nuxeo.web.ui.firstDayOfWeek"></property>
+    
+  </extension>
+</component>
+```
+
+### Others
+By default, there are also two more settings defined related to [Amazon S3 Direct Upload for Web UI]({{page page='amazon-s3-direct-upload'}}) and [Page Providers]({{page page='page-providers'}}).
+The first defines if the S3 Direct Upload is used and the second will define the maximum number of results displayed by the Page Provider.
+
+```xml
+<component name="org.nuxeo.web.ui.properties.contrib">
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+
+     <!-- S3 Direct upload -->
+    <property name="org.nuxeo.web.ui.s3.useDirectUpload">${nuxeo.s3storage.useDirectUpload:=false}</property>
+
+    <!-- Max Results for Tables, Grids and Lists plugged on a Nuxeo Page Provider. Falls back on elasticsearch max result window by default. -->
+    <property name="org.nuxeo.web.ui.listingMaxItems">${org.nuxeo.elasticsearch.provider.maxResultWindow:=10000}</property>
+    
+  </extension>
+</component>
+```
+
+## Customize Settings
+As any other configuration, the best way to create new settings or change/override the default ones, is to contribute a new extension as explained here [HOWTO: Contribute to an Extension]({{page page='how-to-contribute-to-an-extension'}}).
+
+{{#> callout type='warning' }}
+
+Overriding default settings must be done in a conscious way, since it will change the default/expected behavior of the platform, or at least, of some components, depending on which settings are changed.
```

Comment:
```suggestion
Overriding default settings must be done in a conscious way, since it will change the default/expected behavior of the platform or, at least, of some components, depending on which settings are changed.
```